test/govuk_component/related_items_test.rb

```diff
@@ -8,7 +8,6 @@ def component_name
   test "no error if no parameters passed in" do
     assert_nothing_raised do
       render_component({})
-      assert_select ".govuk-related-items"
```

Comment:
Some components use this:
`assert_empty render_component({})`

I think that would be better now.